src/storage/exec/AggregateNode.h

```diff
@@ -35,13 +35,15 @@ struct PropStat {
 template <typename T>
 class AggregateNode : public IterateNode<T> {
  public:
-  using RelNode<T>::execute;
+  using RelNode<T>::doExecute;
 
   AggregateNode(RuntimeContext* context, IterateNode<T>* upstream, EdgeContext* edgeContext)
-      : IterateNode<T>(upstream), context_(context), edgeContext_(edgeContext) {}
+      : IterateNode<T>(upstream), context_(context), edgeContext_(edgeContext) {
+    IterateNode<T>::name_ = "AggregateNode";
```

Comment:
@cangfengzhs 

I really don't recommend this way to initialize the fields of the base class. Please provide the constructor in the base class to do this, such as:

```c++
IterateNode<T>::IterateNode<T>(const std::string& name) : name_(name) {}

AggregateNode(RuntimeContext* context, IterateNode<T>* upstream, EdgeContext* edgeContext) 
  : IterateNode("AggregateNode", upstream), context_(context)...{}
```